client/js/components/month/month-week-board.js

```diff
@@ -1,15 +1,20 @@
 /** @jsx React.DOM */
 var React = require('react');
-var MonthDayCard = require('./month-day-card.js');
+var MonthDayCard = require('./month-day-card');
 var _ = require('underscore');
+var Link = require('react-router-component').Link
 
 var MonthWeekBoard = React.createClass({
   render: function () {
     var weekEvents = _.map(this.props.weekEvents, function(day, key) {
```

Comment:
the variable naming here threw me off a bit - you are mapping weekEvents, so I'd imagine that each item in weekEvents is an event. checking the data, it looks like weekEvents is an array which contains data for each Day (which is well labeled as 'day' in the params). To prevent this confusion later on, I think that we should change the variable 'weekEvents' to something more like 'weekData' which contains 'dayData'. thoughts? 

Also, for the finalized data, I think it'd be clearer for the weekData to be an object with properties for each day with values equal to that days data. Having it in an array just seems a little less robust (we may need to count on the order of the array, etc...)